src/files/delete-prompt/DeletePrompt.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { defineMessages, FormattedMessage } from 'react-intl'
+import TrashIcon from '../../icons/StrokeTrash'
+import CancelIcon from '../../icons/GlyphSmallCancel'
+import Button from '../../components/button/Button'
+
+const messages = defineMessages({
```

Comment:
Is the plan is to move those inline translations to outside files in separate PR?
(as in https://github.com/yahoo/react-intl/wiki#locale-data-in-browsers)